src/routes/ssls/index.tsx

```diff
@@ -1,9 +1,121 @@
-import { createFileRoute } from '@tanstack/react-router'
-
-export const Route = createFileRoute('/ssls/')({
-  component: RouteComponent,
-})
+import { queryClient } from '@/config/global';
+import { useSuspenseQuery } from '@tanstack/react-query';
+import { createFileRoute } from '@tanstack/react-router';
+import { useTranslation } from 'react-i18next';
+import type { APISIXType } from '@/types/schema/apisix';
+import { ProTable } from '@ant-design/pro-components';
+import type { ProColumns } from '@ant-design/pro-components';
+import { useEffect, useMemo } from 'react';
+import PageHeader from '@/components/page/PageHeader';
+import { ToAddPageBtn } from '@/components/page/ToAddPageBtn';
+import { AntdConfigProvider } from '@/config/antdConfigProvider';
+import { usePagination } from '@/utils/usePagination';
+import { pageSearchSchema } from '@/types/schema/pageSearch';
+import { getSSLListQueryOptions } from '@/apis/ssls';
 
 function RouteComponent() {
-  return <div>Hello "/ssl/"!</div>
+  const { t } = useTranslation();
+
+  const { pagination, handlePageChange, updateTotal } = usePagination({
+    queryKey: 'ssls',
+  });
+
+  const sslsQuery = useSuspenseQuery(getSSLListQueryOptions(pagination));
+  const { data, isLoading } = sslsQuery;
+
+  useEffect(() => {
+    if (data?.total) {
+      updateTotal(data.total);
```

Comment:
put `updateTotal` to usePagination will be better.

Reply:
create a issue to trach this, including sort import

Reply:
tracked in #3000